packages/suite-web/e2e/support/utils/shortcuts.ts

```diff
@@ -84,7 +86,21 @@ export const passThroughSetPin = () => {
 
 export const enableDebugMode = () => {
     cy.window().then(window => {
-        window.store.dispatch({ type: '@suite/set-debug-mode', payload: { showDebugMenu: true } });
+        window.store.dispatch({
+            type: SuiteActions.SET_DEBUG_MODE,
+            payload: { showDebugMenu: true },
+        });
+    });
+};
+
+export const disableFirmwareHashCheck = () => {
+    // window.store may not be ready at this point, we need to wait for app to load
+    cy.getTestElement('@welcome-layout/body');
```

Comment:
Idk if waiting on `@welcome-layout/body` is the best way. What I tried:
- waiting on `@device-compromised/hash-check`, but that's not possible because it shows up only for T1B1. I'd need to _optionally_ wait for it, could do that with `MutationObserver` that checks each new node if it might be the error modal, but idk, it's awkward
- `cy.get('[data-testid]')` wait for _any_ test element as proof that React app has rendered. But it's not enough, the[ useCypress()](https://github.com/trezor/trezor-suite/blob/develop/packages/suite-web/src/support/useCypress.ts) has not fired during the _first such render cycle_. It works with `cy.wait(500)` but that's hacky